src/scripts/main.js

```diff
@@ -20,8 +20,32 @@ const food = {
 
 const tree = document.querySelector('#tree');
 
-function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+function createTree(element = document.createElement('ul'), data) {
+  if (!element) {
+    element = document.createElement('ul');
+    element.setAttribute('id', 'tree');
+    document.body.appendChild(element);
+  }
```

Comment:
The check for `!element` is redundant because `element` is always provided when `createTree` is called. You can remove this block to simplify the code.